c7n/policy.py

```diff
@@ -207,6 +213,29 @@ def run(self, *args, **kw):
                 "ActionTime", time.time() - at, "Seconds", Scope="Policy")
             return resources
 
+    def get_logs(self, start, end):
+        log_source = self.policy.ctx.output
```

Comment:
there's a third variant here thats a bit simpler, which is if log group was passed to run and logs, then we can query out from cwl